blocks/library/embed/index.js

```diff
@@ -114,11 +123,15 @@ function getEmbedBlockSettings( { title, icon, category = 'embed', transforms, k
 							return;
 						}
 						response.json().then( ( obj ) => {
-							const { html, type } = obj;
+							const { html, type, provider_name: providerName } = obj;
+							const providerNameSlug = kebabCase( toLower( providerName ) );
+							console.log(providerNameSlug, providerName);
```

Comment:
We'll want to remove this debugging statement.

Reply:
Removed in recent commit.